docs/learning/workshop_a_precision_converter_fpga_integration_journey/index.rst

```diff
@@ -0,0 +1,693 @@
+
+.. figure:: system_build_m2k.png
+   :align: center
+
+- Two programmable power supplies
+- Two-channel USB digital oscilloscope
+- Two-channel arbitrary function generator
+- 16-channel digital logic analyzer (3.3V CMOS and 1.8V or 5V tolerant, 100MS/s)
+
+**System Build - Scopy**
+
+.. figure:: system_build_scopy.png
+   :align: center
+
+Uses the ADALM2000 to implement virtual instruments:
+- Oscilloscope (with Mixed Signal Capability)
+- Signal Generator (Functions and Arbitrary)
+- Spectrum Analyzer
+- Network Analyzer
+- Voltmeter
+- Power Supply
+- Logic Analyzer (with Stack Decoder support)
+- Digital Pattern Generator
+- GPIO
+
+**System Build - Schematic**
+
+.. figure:: system_build_schematic.png
+   :align: center
+
+**System Build - CORA Z7S Configuration**
+
+.. figure:: system_build_cora.png
+   :align: center
+
+**System Build - Power Supply**
+
+.. figure:: system_build_power_supply.png
+   :align: center
+
+**System Build - Input Signal**
+
+.. figure:: system_build_input_signal.png
+   :align: center
+
+**System Build - UART Configuration**
+
+.. figure:: system_build_uart.png
+   :align: center
+
+**System Build – change host IP**
+
+.. figure:: system_build_ip.png
+   :align: center
+
+**System Build - UART and Ethernet Testing**
+
+.. figure:: system_build_uart_and_eth.png
+   :align: center
+
+Evaluate System
+~~~~~~~~~~~~~~~
+
+**System Evaluation – regular SPI trigger configuration**
+
+.. figure:: system_evaluation_spi_trigger_1.png
+   :align: center
+
+.. figure:: system_evaluation_spi_trigger_2.png
+   :align: center
+
+**System Evaluation**
+
+=========================================================== ====================== =====================
+Steps                                                       Regular SPI controller SPI Engine controller
+=========================================================== ====================== =====================
+Connect IIO Oscilloscope and visualize the captured signals
+Run the Python script on Cora z7s / Cygwin
+Analyze results in VisualAnalog
+Compare the results
+=========================================================== ====================== =====================
+
+**System Evaluation - IIO Oscilloscope**
+
+.. figure:: system_evaluation_iio_osc_1.png
+   :align: center
+
+.. figure:: system_evaluation_iio_osc_2.png
+   :align: center
+
+**System Evaluation - Logic Analyzer**
+
+.. figure:: system_evaluation_m2k_1.png
+   :align: center
+
+.. figure:: system_evaluation_m2k_2.png
+   :align: center
+
+**System Evaluation**
+
+=========================================================== ====================== =====================
+Steps                                                       Regular SPI controller SPI Engine controller
+=========================================================== ====================== =====================
+Connect IIO Oscilloscope and visualize the captured signals           X
+Run the Python script on Cora z7s / Cygwin
```

Comment:
- [ ] When referring to Cora, please do it like this:

```suggestion
Run the Python script on `Cora Z7S`_ / Cygwin
```

- [ ] And on the last line from the file, add this line:

.. _Cora Z7S: https://digilent.com/shop/cora-z7-zynq-7000-single-core-for-arm-fpga-soc-development